building_applications/building_apps_define_manifest.md

```diff
@@ -16,116 +16,23 @@ The manifest file can also define what permissions and access rights your App re
-
-    ![New Manifest Template]({{ site.baseurl }}/assets/guides/create-new-manifest-v4.1-install-instructions.png)
-
-5. Refer to the next sections for adding more components to the manifest.
-
-### Adding Embedded Fullscreen Components
-
-You can add a fullscreen embedded component to the manifest using the **Inject Component** button for the `embedded` component type.
-
-![Inject Embedded]({{ site.baseurl }}/assets/guides/inject-component-embedded.png)
-
-A new code block is added to the manifest where you can define the structure of your embedded (fullscreen) component.
-For example:
-
-![Embedded Snippet]({{ site.baseurl }}/assets/guides/create-new-manifest-v4.1-fullscreen-snippet.png)
-
-The manifest objects and attributes for embedded component are described below.
-
-| __Manifest Object/Attribute__ | __Description__ |
-| `desciption` | Used for entering a meaningful text description for the component. |
-| `source_url` | Defines the URL for the component and defines any parameter interpolation required for your application. |
-| `user_configuration:schema` | Defines the `properties` for the component and specifies which ones are required. |
-| `user_configuration:schema:properties` | Defines one or more property objects used for gathering user configuration input. The `name` and `description` attribute values are displayed to the user as configurable fields. |
-
-See [Building Procore Embedded (Fullscreen) Applications]({{ site.url }}{{ site.baseurl }}{% link building_applications/building_embedded_fullscreen_apps.md %}) for additional information.
-
-### Adding Side Panel Components
-
-You can add a side panel component to the manifest using the **Inject Component** button for the `sidepanel` component type.
-
-![Inject SidePanel]({{ site.baseurl }}/assets/guides/inject-component-sidepanel.png)
-
-A new code block is added to the manifest where you can define the structure of your side panel component. For example:
-
-![Sidepanel Snippet]({{ site.baseurl }}/assets/guides/create-new-manifest-v4.1-sidepanel-snippet.png)
-
-The manifest objects and attributes for the side panel component are same as for the embedded component, with the addition of the `sidepanel:views` array.
-This is where you define which Procore application views your component supports.
-For example, `"commitments.purchase_orders.detail"` and `"commitments.purchase_orders.edit"`.
-See [Building Procore Side Panel Applications]({{ site.url }}{{ site.baseurl }}{% link building_applications/building_side_panel_apps.md %}) for additional information on supported views.
-
-### Adding Data Connection Components
-
-You can add a data connection component to the manifest using the **Open Builder** button under App Tool Permissions.
+When you create a new application on the developer portal, the system first generates a sandbox environment for your new application, and then directs you to the Manage App page.
```

Comment:
Can we make a 'plug' that hints that the sandbox environment is their primary environment for testing/learning about Procore?

As an example - "the system first generates a sandbox environment for your new application, which is intended for application development and testing...."

We are trying to push all developers to test or learn about Procore in the sandbox environment as they currently ask for production instances to test/develop in.

Reply:
For consistency, should we reference development sandbox since there are multiple sandbox enviornments?